pyregion/conftest.py

```diff
@@ -2,7 +2,19 @@
 # by importing them here in conftest.py they are discoverable by py.test
 # no matter how it is invoked within the source tree.
 
-from astropy.tests.pytest_plugins import *
+from astropy.version import version as astropy_version
+if astropy_version < '3.0':
```

Comment:
this won't be compatible with astropy <2.0.3, but I think that's OK (run into it in astroquery)